README.md

```diff
@@ -17,8 +27,57 @@ For more information on Fixtures, check out this [pytest documentation](https://
 A **Mock** is an object that simulates the behavior of another object such as a class or function. They are used to simulate the behavior of real objects for testing purposes.
 For more information on Mocks, check out this [unittest documentation](https://docs.python.org/3/library/unittest.mock.html) on the mock module.
 
-**Example:**
+## Installation
+
+To use the **JupyterHub Pytest Plugin**, you will first need to install it using pip by either:
+
+- installing it locally
```

Comment:
I believe it's worth leaving a note here about the fact that we're instructing to install the package from GitHub because the package was not yet released on PyPI. And then change the install instruction once that's done. Also, adding this to a todo list, somewhere in an issue to not forget, will be also a good idea.